modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -91,7 +92,8 @@ class ImagingUploader extends Component {
    */
   formatColumn(column, cell, rowData, rowHeaders) {
     // If a column if set as hidden, don't display it
-    if (this.state.hiddenHeaders.indexOf(column) > -1) {
+    if (loris.hiddenHeaders !== undefined &&
```

Comment:
i think this.state.hiddenHeaders can be called here instead of loris.hiddenHeaders so as not to confuse it with the actual global loris variable